accounts/usbwallet/ledger.go

```diff
@@ -208,7 +215,30 @@ func (w *ledgerDriver) SignPersonalMessage(path accounts.DerivationPath, message
 	return w.ledgerSignData(path, message)
 }
 
-// ledgerVersion retrieves the current version of the Celo wallet app running
+/* TODO: add functionality to the Ledger's Celo app
```

Comment:
where do these comments on this file come from?

Reply:
@gastonponti

Reply:
Yes. The thing here is that ethereum changed their app to use this from the ledger.
In our case, I'm pretty sure that we can't handle this. So I basically commented this to avoid errors, until we can support that.
I'll ping the crypto team, but I don't think that this should be a limitation for this PR to be merged. It's something that we can enable in a future version

Reply:
can we create an issue on adding support for this to the ledger?